### PR TITLE
added instructions for reviewers for level 1 coding challenges

### LINKED ISSUE #103: Create "##Instructions for reviewer" for project Level 1 Programming Katas
Please add a section to the bottom of this project's _index.md file.
The section heading should be EXACTLY "## Instructions for reviewer"

Basically we have to set reviewers up for success by telling them what success looks like and what to pay attention to when reviewing this project submission.

For this project, One of the most common problems is merge conflicts. There should be a point that says "If there are merge conflicts then this work is not yet competent" in your instructions.
Description:

- [ ]  Look at a bunch of old reviews for this project for each cohort

- [ ]  Find the common feedback [This should help you get an idea of what instructions to create]

- [ ]  Write them out clearly

- [ ]  The section heading should be EXACTLY "## Instructions for reviewer"

### PR DESCRIPTION
**Pull request by Lisa1400**
Related issues: #103 

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by ry-oc — APPROVED** [review]

File: content/projects/katas/level-1/_index.md [hunk 1]
```diff
@@ -177,4 +177,47 @@ eg `combine([11,22,33], [1,2,3])` should return
 
 ```
 [11,1,22,2,33,3]
-```
\ No newline at end of file
+```
+
+## Instructions for Reviewers
```
Comment (ry-oc) [inline content/projects/katas/level-1/_index.md:182]: This is a bit too hi-res and you are repeating many of the instructions that are listed above, lets get this cleaned up :)

File: content/projects/katas/level-1/_index.md [hunk 2]
```diff
@@ -177,4 +177,47 @@ eg `combine([11,22,33], [1,2,3])` should return
...
+**Task 1.1**
+
+- The instructions does not specify whether a value should be returned or printed. Therefore, either is competent.
+- The instructions also does not specify whether a function should be created. Therefore, if a function was created or only lines of code were written and it works, it is competent.
+- When a 1000 is the input, ensure that the output is 233168.
+
+**Task 1.2 and Task 1.3**
+
+- Ensure that the function takes in two integer arguments.
+- Ensure that the logical operators are correct.
```
Comment (ry-oc) [inline content/projects/katas/level-1/_index.md]: 196 & 197 are repeats of the instructions above

File: content/projects/katas/level-1/_index.md [hunk 3]
```diff
@@ -177,4 +177,47 @@ eg `combine([11,22,33], [1,2,3])` should return
...
+- Ensure that keywords(**not strings**) true or false are being returned.
+
+**Task 1.4**
+
+- Ensure that the function takes in an integer argument.
+- The number that you input should output that amount of hashes in rows and columns.
```
Comment (ry-oc) [inline content/projects/katas/level-1/_index.md]: This is also listed in the instructions above

File: content/projects/katas/level-1/_index.md [hunk 4]
```diff
@@ -177,4 +177,47 @@ eg `combine([11,22,33], [1,2,3])` should return
...
+
+
+**Task 1.5**
+
+- Ensure that the function takes in an integer argument.
+- When a positive number is the input, a right-angled triangle of hashes should be the output.
+- When a negative number is the input, an upside-down right-angled triangle of hashes should be the output.
+- All 3 sides of the triangle should the same amount as the input.
```
Comment (ry-oc) [inline content/projects/katas/level-1/_index.md]: Some of these are also repeats

File: content/projects/katas/level-1/_index.md [hunk 5]
```diff
@@ -177,4 +177,47 @@ eg `combine([11,22,33], [1,2,3])` should return
...
+
+**Task 1.6**
+
+- Ensure that an **array of strings** is being passed as an argument.
```
Comment (ry-oc) [inline content/projects/katas/level-1/_index.md]: This line is a repeat

File: content/projects/katas/level-1/_index.md [hunk 6]
```diff
@@ -177,4 +177,47 @@ eg `combine([11,22,33], [1,2,3])` should return
...
+- The function should take in any amount of strings.
+- If there are strings with the same length, all should be printed in "next line" form.
+
+**Task 1.7**
+
+- Ensure that two arrays are being passed as arguments.
```
Comment (ry-oc) [inline content/projects/katas/level-1/_index.md]: remove the repeated instructions.

**Review by ry-oc — CHANGES_REQUESTED** [review]
Hey, so a bunch of stuff is being repeated here. If the instruction exists in the project instruction then we can rely on the reviewer to read the instructions before reviewing

File: content/projects/katas/level-1/_index.md [hunk 7]
```diff
@@ -177,4 +177,31 @@ eg `combine([11,22,33], [1,2,3])` should return
...
+
+- The instructions does not specify whether a value should be returned or printed. Therefore, either is competent.
+- The instructions also does not specify whether a function should be created. Therefore, if a function was created or only lines of code were written and it works, it is competent.
+- When a 1000 is the input, ensure that the output is 233168.
+
+**Task 1.2 and Task 1.3** 
+
+- Ensure that keywords(**not strings**) true or false are being returned.
+
+**Task 1.4 and Task 1.5** is pretty straight forward.
```
Comment (ry-oc) [inline content/projects/katas/level-1/_index.md]: Does this line serve a purpose or provide any additional info?

File: content/projects/katas/level-1/_index.md [hunk 8]
```diff
@@ -177,4 +177,31 @@ eg `combine([11,22,33], [1,2,3])` should return
...
 ```
 [11,1,22,2,33,3]
-```
\ No newline at end of file
+```
+
+## Instructions for Reviewers
+
+- Ensure that every task has its own file, and branch.
+- Check if a specific task wants an output to be returned, if so the **return** keyword should be used to stop the execution of the function.
```
Comment (ry-oc) [inline content/projects/katas/level-1/_index.md]: RE: 184, 185, & 186
Does this even need to be said at all? I think this is in the project instructions above as well? Things should be DRY

**Review by ry-oc — CHANGES_REQUESTED** [review]
Some small things

**Review by ry-oc — APPROVED** [review]
looks good to me

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**